examples/experimental/i3s-picking/luma/gltf-material-parser.js

```diff
@@ -75,14 +74,22 @@ export default class GLTFMaterialParser {
         gltfTexture.texture.sampler.parameters) ||
       {};
 
+    const image = gltfTexture.texture.source.image;
```

Comment:
Fine for now, though I would prefer luma.gl Texture to just accept compressed texture as `data` field without any switch cases.

But I want to make a pass on defining the exact structure of the data that is returned from the texture loaders before we publish 3.0, so don't make any additional changes yet.